pom.xml

```diff
@@ -19,6 +19,11 @@
         <!--examples-->
         <module>examples/processors</module>
         <module>docker</module>
+        <!-- test jar builders -->
+        <module>streams/service/src/test/resources/custom-udf-microtest</module>
```

Comment:
Streamline-functions builds its own target jar, so it can't have a subsidiary module as a jar. 
 So I put the test jar sources under streamline/test-support/custom-udf-microtest.  

If folks want to similarly move source code for other test jars, like those in streams/service/src/test/resources/customprocessorupload, into test-support/, it's a good place that avoids dependency ordering problems.